### PR TITLE
expose getTransactionResultByIndex

### PR DESCRIPTION
**Pull request by bjartek**
Expose the grpc method to get TransactionResultByIndex since it is the only way to get systemChunkEvents and not get all other transactions in the same block

### CODE REVIEW

File: access/grpc/grpc.go [hunk 1]
```diff
@@ -320,6 +320,31 @@ func (c *BaseClient) GetTransactionResult(
...
 }
 
+func (c *BaseClient) GetTransactionResultByIndex(
+	ctx context.Context,
+	blockID flow.Identifier,
+	index uint32,
+	opts ...grpc.CallOption,
+) (*flow.TransactionResult, error) {
+
+	//GetTransactionResultByIndex(ctx context.Context, in *GetTransactionByIndexRequest, opts ...grpc.CallOption) (*TransactionResultResponse, error)
```
Comment (nvdtf) [inline access/grpc/grpc.go:330]: extra line?

Comment (bjartek) [inline access/grpc/grpc.go:330]:
```suggestion
```

Comment (bjartek) [reply access/grpc/grpc.go]: fixed

**Review by janezpodhostnik — APPROVED** [review]

**Review by nvdtf — APPROVED** [review]

**Comment by janezpodhostnik** [comment]
@bjartek can you update this PR?

**Comment by bjartek** [comment]
updated, sorry was away for a bit

**Review by sideninja — APPROVED** [review]

**Pull request merged**